MSAL/src/public/MSALAccount.h

```diff
@@ -89,6 +90,32 @@
   */
 @interface MSALAccount : NSObject <MSALAccount, NSCopying>
 
+#pragma mark - Getting information about account in different AAD tenants
+
+/**
+ Array of all tenants for which a token has been requested by the client.
+ 
+ @note This field will only be available when querying account(s) by the following APIs of MSALPublicClientApplication:
+ `-[MSALPublicClientApplication allAccounts:]`
+ `-[MSALPublicClientApplication accountForUsername:error:]`
+ 
+ The field will be nil in other scenarios. E.g., account returned as part of the result of an acquire token interactive/silent call.
+ */
+@property (readonly, nullable) NSArray<MSALTenantProfile *> *tenantProfiles;
```

Comment:
these properties needs to be declared in the "@protocol" and then "implemented in the "MSALAccount+Internal.h"

Reply:
Updated. I have followed the example of @property (readonly, nullable) NSString *username as suggested